### DISCUSSION TITLE
TOTP Setup: No Indication of Allowed Periods

### DISCUSSION THREAD

**Discussion by ghost** [authelia/authelia#6456] [Pre-Release Feedback]
Sorry for the bad title...wasn't quite sure how to word it, but here's a picture:

![image](https://github.com/authelia/authelia/assets/113205165/c558238f-5e5f-4ae0-9a0d-44764b4bee71)

As you can see, there's the selector for 30...but no title, like `Algorithm` and `Length` above.

**Comment by james-d-elliott** [comment]
Should be fixed if you can confirm (will be pushed in 5 mins), just didn't hide the radio itself, was hiding the group and the label only lol.

**Reply by ghost** [reply]
Unfortunately, I still just see `30` and no label above it.

Edit: I've tried both with `totp.allowed_periods: [ 30 ]` defined and no `totp.allowed_periods` defined. (Of course restarting Authelia for the new config changes and launching new private/Tor browser windows.

**Comment by james-d-elliott** [comment]
Fixed

**Reply by ghost** [reply]
Confirmed. :)

**Reply by ghost** [reply]
Also: Thanks and happy holidays and such. :)
